frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -632,23 +633,18 @@ class DatasetImportView extends React.PureComponent<Props, State> {
     if (!form) {
       return;
     }
-
-    const parsedConfig = JSON.parse(form.getFieldValue("dataSourceJson"));
     if (syncTargetTabKey === "advanced") {
       // Copy from simple to advanced: update json
 
-      // parsedConfig has to be used as the base, since `dataSource` will only
-      // contain the fields that antd has registered input elements for
```

Comment:
I doubletested that this comment is indeed outdated. Opening the edit-view for a dataset and changing information in the basic tab, still has the complete JSON available when syncing the tabs :+1: